src/views/LoginSuccess.vue

```diff
@@ -229,6 +229,14 @@ export default class LoginSuccess extends Vue {
         if (this.accountsUpdateCount) return Array.from(this.accounts.values());
         return [];
     }
+
+    private get appName() {
+        console.log(staticStore);
+        if (staticStore.originalRequestType) {
+            return staticStore.originalRequestType;
```

Comment:
This does not seem right.
The request types are not necessarily good language for display, neither are the routes. Also might cause some trouble when translating.